### ISSUE TITLE
Segmentation Fault when loading --score file with missing value in weight column

### ISSUE THREAD

**Issue by sritchie73** [chrchang/plink-ng#122]
I was getting segmentation faults upon loading a --score file which I eventually tracked down to the presence of NA values in the weights column upon loading the score file into R. Examining the score file itself, it was tab separated, the weight column was the final column in the file, and these entries were single spaces after the final tab separator at the end of the line.

**Comment by chrchang** [comment]
Can you post an example and .log file to replicate this with?

**Comment by sritchie73** [comment]
I've attached a small toy example bad score with 10 variants that should replicate the below log: [bad.txt](https://github.com/chrchang/plink-ng/files/3453401/bad.txt)

```
PLINK v2.00a2LM AVX2 Intel (24 Jul 2019)       www.cog-genomics.org/plink/2.0/
(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3
Logging to test.log.
Options in effect:
  --bfile chr15
  --memory 25000
  -- test
  --score bad.txt 1 4 6 header list-variants cols=nmissallele,scoresums
  --threads 1

Start time: Wed Jul 31 19:39:50 2019
191898 MiB RAM detected; reserving 25000 MiB for main workspace.
Using 1 compute thread.
43059 samples (0 females, 0 males, 43059 ambiguous; 43059 founders) loaded from
chr15.fam.
1604439 variants loaded from chr15.bim.
Note: No phenotype data present.
Calculating allele frequencies... done.
Segmentation fault
```

When the score file is larger, the Segmentation fault message occurs immediately after `--score: 10K variants loaded`

**Comment by chrchang** [comment]
Thanks; this should be fixed by 20ddcc8 .
